src/nerve/core/nodes/identity.py

```
@@ -0,0 +1,151 @@
+            "output": output,
+            "input": output,  # For compatibility with :::N['input'] in Commander
+            "error": None,
+        }
+
+    async def interrupt(self) -> None:
+        """No-op for IdentityNode.
+
+        IdentityNode execution is instantaneous, so interrupt does nothing.
+        This method exists to satisfy the Node protocol.
+        """
+        pass
```

Ruff (0.14.10)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)